eliot/_validation.py

```diff
@@ -156,16 +156,19 @@ def validate(value):
 
 
 
-def fields(**keys):
+def fields(*fields, **keys):
     """
     Factory for for L{MessageType} and L{ActionType} field definitions.
 
+    @param *fields: A L{tuple} of L{Field} instances.
+
     @param **keys: A L{dict} mapping key names to the expected type of the
         field's values.
 
     @return: A L{list} of L{Field} instances.
     """
-    return [Field.forTypes(key, [value], "") for key, value in keys.items()]
+    return list(fields) + [
+        Field.forTypes(key, [value], "") for key, value in keys.items()]
```

Comment:
@itamarst Looks like `_MessageSerializer` does duplicate key detection <https://github.com/ClusterHQ/eliot/blob/master/eliot/_validation.py#L190> there's also a test for it: `MessageSerializerTests.test_noMultipleFields`.

How would you suggest ensuring that `*fields` contains only `Field`s? `isinstance` checks? `_MessageSerializer` will raise an exception trying to access a nonexistent attribute if you're passing something that doesn't look like a `Field` but what's the difference between `MessageType(u'key', [42])` and `fields(42)`?